hermes-mock/src/test/groovy/pl/allegro/tech/hermes/mock/HermesMockAvroTest.groovy

```diff
@@ -71,6 +72,39 @@ class HermesMockAvroTest extends Specification {
             response2.status == HttpStatus.SC_CREATED
     }
 
+    def "should receive an Avro message matched by pattern different nullable schema"() {
+        given: "define wiremock responses for 2 topics with different schemas"
+            def topicName = "my-test-avro-topic-1"
+            def topicName2 = "my-test-avro-topic-2"
+
+
+            hermes.define().avroTopic(topicName,
+                    aResponse().withStatusCode(201).build(),
+                    schema,
+                    TestMessage,
+                    { it -> it.key == "test-key-pattern1" })
+            hermes.define().avroTopic(topicName2,
+                    aResponse().withStatusCode(201).build(),
+                    differentNullableSchema,
+                    TestMessageWithNullableSchema,
+                    { it -> it.value.contains(7) })
+
+
+        when: "messages with matching patterns are published on topics"
+            def message = new TestMessage("test-key-pattern1", "test-key-value")
+            def message2 = new TestMessageWithNullableSchema("test-key-name", 7)
+
+            def response = publish(topicName, message)
+            def response2 = publish(topicName2, message2)
+
+        then: "check for any single message on the topics and check for correct responses"
+            hermes.expect().singleMessageOnTopic(topicName)
+            hermes.expect().singleMessageOnTopic(topicName2)
+
+            response.status == HttpStatus.SC_CREATED
+            response2.status == HttpStatus.SC_CREATED
+    }
+
```

Comment:
**nitpick:** empty blank line

You can remove it to match file formatting style (1 empty line between sections).

Reply:
done